Update to support another implementation with Blazor where messages d…

R4ML1N opened the pull request:
Hi! First of all thanks for your work with this extension. In case it helps, here are some changes I made first to be able to compile the app with the latest version of the montoya-api. Then I made some other changes to support a Blazor APP that I recently ran into. This other app does not follow the same approach of including the *_blazor?id* as part query string in each request. But instead it included the SignalR header in each request for example. Therefore, I used that to recognize the messages and enable the parsing. 

Thanks!

R4ML1N committed:
Update to support another implementation with Blazor where messages does not include *blazor?id* in the path.

irsdl commented:
This sounds like a good update, I am going to fork the PR instead of the main project now!

irsdl committed:
WebSocket and bug fixes

Adding web socket support
Adding a better rule to exclude messages which are not serialized in blazer during the handshakes
Displaying an error message when a message is corrupted or incomplete
Helper function code improvements
Upgrading Gradle libraries to their latest
Using Java 19
Using the latest Montoya version on 20/10/2023
Optimized library imports using intelliJ
Extension version increased to 1.1
Added github workflows to compile it automatically and save it in the /releases/ directory (needs testing in practice)
This is based on a fork of https://github.com/AonCyberLabs/BlazorTrafficProcessor/tree/main

irsdl committed:
Update .gitignore

irsdl committed:
Update build.gradle

irsdl committed:
Update gradle.yml

irsdl committed:
Update gradle.yml

irsdl committed:
Update gradle.yml

irsdl committed:
Update gradle.yml

irsdl committed:
Update README.md

Commit:
Make downgrade options configurable

Commit:
Keep options panel open to select multiple

Commit:
Merge branch 'pr-2' into my-integration

Commit:
Merge branch 'pr-3' into my-integration

Commit:
Revert "Merge branch 'pr-3' into my-integration"

This reverts commit c215c1fb7f3f525d48aa59f50e2100f68c776562, reversing
changes made to daa25913ee5c74f39fed5fd382e969cbfd005c2e.

Commit:
Adding DockerFile and updating build.gradle file, some changes to remove old library also.

R4ML1N committed:
Merge pull request #1 from R4ML1N/my-integration

Integrate upstream PRs

R4ML1N committed:
Changes and Fixes

R4ML1N committed:
Changes and Fixes

R4ML1N committed:
Updated README

R4ML1N committed:
Pipeline fix

R4ML1N committed:
Pipeline fix

R4ML1N committed:
Updated Scope Logic Check

R4ML1N committed:
Updated Readme.md
